File: beginner/javascript/1066.js
var input = require('fs').readFileSync('stdin', 'utf8');

var valores = input.split('\n');
var n, i, contPar = 0, contImpar = 0, contPos = 0, contNeg = 0;

for(i = 0; i < 5; i++){
    n = parseInt(valores.shift());
    if(n > 0)
        contPos+=1;
    if(n < 0)
        contNeg+=1;
    if(n % 2 == 0)
        contPar+=1;
    if(n % 2 != 0)
        contImpar+=1;
}

console.log(contPar + " valor(es) par(es)");
console.log(contImpar + " valor(es) impar(es)");
console.log(contPos + " valor(es) positivo(s)");
console.log(contNeg + " valor(es) negativo(s)");

File: beginner/python/1066.py
contPos = 0
contNeg = 0
contPar = 0
contImpar = 0

for i in range(0,5):
        n = int(input())
        if(n > 0):
            contPos+=1
        if(n < 0):
            contNeg+=1
        if(n % 2 == 0):
            contPar+=1
        if(n % 2 != 0):
            contImpar+=1
    

print("%d valor(es) par(es)" % contPar)
print("%d valor(es) impar(es)" % contImpar)
print("%d valor(es) positivo(s)" % contPos)
print("%d valor(es) negativo(s)" % contNeg)